varya/style-rtl.css

```diff
@@ -3271,6 +3271,7 @@ nav a {
 
 .entry-meta a,
 .entry-footer a {
+	border-bottom: none;
```

Comment:
This appears to be leftover from a previous commit. But I think it's supposed to be in here.

Reply:
Yep, noticed this as well. Thanks for checking it in.